cli/cmd/publicendpoint.go

```diff
@@ -0,0 +1,379 @@
+// Copyright 2016 The Serviced Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"encoding/json"
+    "fmt"
+	"os"
+    "strconv"
```

Comment:
Again - more tab/space inconsistencies. The convention here for GO code is to indent with tab characters.

Also, you can try using 'gofmt' to format new files.